inst/WORDLIST

```diff
@@ -24,3 +24,12 @@ responders
 Teckla
 tte
 www
+ADSL
+ANLzzFL
+AVALC
+bor
+BoR
+param
+PARCATy
+RSSTRES
+RSSTRESC
```

Comment:
Are these entries really required? Backticks should be used for variable names.

Reply:
Ive removed the duplicates, the others I have in the documentation

Reply:
Please remove the variable names. Text in backticks is not check for spelling errors. By removing the variable names from the wordlist we can detect if there are backticks missing.

Reply:
Oh I see, I had not actually realised this, what is the spelling package we use in CI so I can check before I push?